src/Microsoft.ML.Ensemble/OutputCombiners/MultiStacking.cs

```diff
@@ -57,7 +57,7 @@ public Arguments()
                     env => new Ova(env, new Ova.Arguments()
                     {
                         PredictorType = ComponentFactoryUtils.CreateFromFunction(
-                             e => new AveragedPerceptronTrainer(e, new AveragedPerceptronTrainer.Arguments()))
+                            e => new FastTreeBinaryClassificationTrainer(e, DefaultColumnNames.Label, DefaultColumnNames.Features))
```

Comment:
>FastTreeBinaryClassificationTrainer [](start = 37, length = 35)

I'd really rather we didn't. This seems to fit into the same bucket as the discussion on #682. That ensembling should have a dependency on FastTree merely because we have a default does not make sense to me. If someone wants to use stacking, that's great, but they need to specify the learners.

Reply:
But maybe we can hold off for right now.

---
In reply to: [218215145](https://github.com/dotnet/machinelearning/pull/855#discussion_r218215145) [](ancestors = 218215145)

Reply:
Yes, let's do that separately, when we shape the ensembles. 

---
In reply to: [218215323](https://github.com/dotnet/machinelearning/pull/855#discussion_r218215323) [](ancestors = 218215323,218215145)